doc/deoplete.txt

```diff
@@ -516,8 +516,12 @@ EXAMPLES						*deoplete-examples*
 SOURCES							*deoplete-sources*
 
 buffer						*deoplete-source-buffer*
-		It collects keywords from current buffer and the buffers which
-		have same 'filetype'.
+		It collects keywords from current buffer and buffers which
+		have same 'filetype', unless you specify this custom setting:
+>
+		" Collect keywords from all buffers, regardless of filetype.
+		call deoplete#custom#set('buffer', 'require_same_filetype', 0)
```

Comment:
It does not work.
Have you tested it?

It should be `g:deoplete#buffer_require_same_filetype` variable.